client/components/site-icon/style.scss

```diff
@@ -9,13 +5,13 @@
 	overflow: hidden;
 	align-self: center;
 	margin: 0;
+	text-align: center;
 
 	// Globe icon for sites without an icon
 	&.is-blank {
-		.noticon {
+		.gridicon {
 			color: $white;
-			text-align: center;
-			z-index: z-index( 'root', '.site-icon.is-blank .noticon' );
+			z-index: z-index( 'root', '.site-icon.is-blank .gridicon' );
```

Comment:
We need to update this in the [z-index mapping](https://github.com/Automattic/wp-calypso/blob/1d0480ea45da81f3c06f46abf4844f0cb080058d/assets/stylesheets/shared/functions/_z-index.scss#L38).